content/en/blogs/transform-traditional-applications-into-microservices.md

```diff
@@ -0,0 +1,227 @@
+---
+title: 'Refactor Traditional Applications into Microservices to Enable the Traffic Monitoring Feature'  
```

Comment:
`Transform` seems more commonly used than `Refactor ` from my investigation. `Transform` is much relevant to this blog.